napari/_qt/qt_event_loop.py

```diff
@@ -176,6 +181,8 @@ def quit_app():
 def gui_qt(*, startup_logo=False, gui_exceptions=False, force=False):
     """Start a Qt event loop in which to run the application.
 
+    NOTE: This method will soon be deprecated. Please use ``napari.run()`` instead.
```

Comment:
Oof. I'm supportive of the initiative, but this is a really major API for us, as big as `add_image`. I'm getting jitters about using the d-word so definitively here. What do you think about 

```
This context manager may be deprecated in the future. Prefer using
:py:func:`napari.run` instead.
```

?

Reply:
softened the language... though, I'm not sure why, since I thought we all agreed that we _are_ deprecating it soon.  and obviously, deprecating doesn't mean breaking... so no need for jitters :joy: